ext/filter/filter.stub.php

```diff
@@ -2,16 +2,16 @@
 
 /** @generate-function-entries */
 
-function filter_has_var(int $type, string $variable_name): bool {}
+function filter_has_var(int $type, string $var_name): bool {}
```

Comment:
since we are talking about a superglobal array key... Could we use `$key`? Yeah, I admit that it might be a bit too unorthodox a name.

Reply:
```suggestion
function filter_has_var(int $input_type, string $var_name): bool {}
```
I had to look at the docs what this means. And I think being explicit with `$var_name` is good here ...